pkg/projects/Microsoft.NETCore.App/project.json

```diff
@@ -3,22 +3,22 @@
     "Microsoft.CodeAnalysis.CSharp": "1.3.0",
     "Microsoft.CodeAnalysis.VisualBasic": "1.3.0",
     "Microsoft.CSharp": "4.0.1",
-    "Microsoft.NETCore.Runtime.CoreCLR": "1.0.2",
+    "Microsoft.NETCore.Runtime.CoreCLR": "1.0.4-beta-24403-05",
```

Comment:
Any reason you are picking build 24403-05? instead of the latest build? 

If you take the latest build you are going to need to update the version to 1.1.0 for the CoreCLR package.